app/connectors/PreferencesConnector.scala

```
@@ -37,17 +36,17 @@ class PreferencesConnector @Inject()(httpClient: HttpClient, val runModeConfigur
 
   lazy val preferencesBaseUrl: String = baseUrl("preferences")
 
-  def getPreferredEmail(nino: String, defaultEmail: String)(implicit headerCarrier: HeaderCarrier): Future[String] = {
+  def getPreferredEmail(nino: String)(implicit headerCarrier: HeaderCarrier): Future[String] = {
     try {
       for {
         entityId <- entityResolverConnector.resolveEntityIdFromNino(Nino(nino))
         email <- httpClient
           .GET[HttpResponse](s"$preferencesBaseUrl/preferences/$entityId")
           .map(e => (Json.parse(e.body) \ "email" \ "email").as[String])
-          .recover({ case _ => defaultEmail})
+          .recover({ case _ => ""})
       } yield email
     } catch {
-      case _: uk.gov.hmrc.http.NotFoundException => Future.successful(defaultEmail)
+      case _: Throwable => Future.successful("")
```

Do we really need to ignore all errors here?

---

Replied offline